controllers/comments-controller.js

```diff
@@ -21,3 +24,22 @@ exports.getCommentsById = (req, res, next) => {
       next(err);
     });
 };
+
+// TASK 10
+exports.postCommentsById = (req, res, next) => {
+  const id = req.params.review_id;
+  const newComment = req.body;
+
+  const promises = [
+    checkValExists("reviews", "review_id", id),
+    checkValExists("reviews", "owner", req.body.username),
+    insertCommentsById(id, newComment),
+  ];
+  return Promise.all(promises)
+    .then((data) => {
+      res.status(201).send({ comment: data[2] });
+    })
+    .catch((err) => {
+      next(err);
+    });
+};
```

Comment:
Do we need to be checking if the property exists before we try and post here or is trying to post to a non-existant endpoint something that we can simply handle as a PSQL error?